tests/tests_app/utilities/test_tree.py

```diff
@@ -35,74 +35,6 @@ def __init__(self):
         self.mixed_right = SimpleFlowTree()
 
 
-@pytest.mark.parametrize(
-    "input_tree, types,expected_sequence",
-    [
-        (LeafFlow(), (LightningFlow,), ["root"]),
-        (LeafWork(), (LightningFlow,), []),
-        (
-            SimpleFlowTree(),
-            (LightningFlow,),
-            [
-                "root",
-                "root.simple_flow_right",
-                "root.simple_flow_left",
-            ],
-        ),
-        (SimpleWorkTree(), (LightningFlow,), ["root"]),
-        (
-            SimpleWorkTree(),
-            (LightningFlow, LightningWork),
-            [
-                "root",
-                "root.simple_work_right",
-                "root.simple_work_left",
-            ],
-        ),
-        (
-            MixedTree(),
-            (LightningFlow,),
-            [
-                "root",
-                "root.work_tree",
-                "root.mixed_right",
-                "root.mixed_right.simple_flow_right",
-                "root.mixed_right.simple_flow_left",
-                "root.mixed_left",
-                "root.mixed_left.simple_flow_right",
-                "root.mixed_left.simple_flow_left",
-            ],
-        ),
-        (
-            MixedTree(),
-            (LightningWork,),
-            [
-                "root.work_tree.simple_work_right",
-                "root.work_tree.simple_work_left",
-            ],
-        ),
-        (
-            MixedTree(),
-            (LightningFlow, LightningWork),
-            [
-                "root",
-                "root.work_tree",
-                "root.work_tree.simple_work_right",
-                "root.work_tree.simple_work_left",
-                "root.mixed_right",
-                "root.mixed_right.simple_flow_right",
-                "root.mixed_right.simple_flow_left",
-                "root.mixed_left",
-                "root.mixed_left.simple_flow_right",
-                "root.mixed_left.simple_flow_left",
-            ],
-        ),
-    ],
-)
-def test_depth_first(input_tree, types, expected_sequence):
```

Comment:
reviewer: removed because depth_first is unused

Reply:
Yes.